components/faq/questions.js

```diff
@@ -0,0 +1,227 @@
+import React from 'react';
+import OutBoundLink from 'common/components/OutboundLink/OutboundLink';
+import { donateLink } from 'common/constants/urls';
+
+const questions = () => ({
+  general: [
+    {
+      question: 'When was Operation Code founded?',
+      answer:
+        'The first line of code for the Operation Code site was pushed on August 21, 2014. ' +
+        'Operation Code formally became a 501(c)3 nonprofit on May 4, 2016.',
+    },
+    {
+      question: 'How was Operation Code founded?',
+      answer:
+        'Operation Code was founded in Portland, Oregon by ex-Army Captain David Molina,' +
+        ' who took action and built operationcode.org to petition Congress to expand the' +
+        ' New GI Bill to include code schools.',
+    },
+    {
+      question: 'Is Operation Code a 501(c)(3) nonprofit organization?',
+      answer:
+        'Yes. Operation Code was granted tax-exempt status by the Internal Revenue Service,' +
+        ' functioning as a charity under 501(c)(3) of the Internal Revenue Code, effective 4' +
+        ' May 2016, with retroactive status to 15 June 2015. Donations to the organization are' +
+        ' deductible as charitable contributions.',
+    },
+    {
+      question: 'Where is Operation Code based? Do you have a location near me?',
+      answer:
+        'Operation Code, much like software, is built from anywhere with an internet connection,' +
+        " and is not based in one location. While we're headquartered in Portland, the entire" +
+        ' organization is decentralized, including the board of directors and the core team. This' +
+        " allows us to more effectively serve the entire military community, whether they're" +
+        " veterans or military spouses, whether they're OCONUS or in-country. We have chapters" +
+        ' all over the nation. Use Slack chat and join the closest town to you!',
+    },
+    {
+      question: 'Who does Operation Code serve?',
+      answer:
+        "Operation Code serves our nation's finest who've worn the uniform and their families" +
+        ' who are interested in coding and software development. Our programs are offered at no' +
+        ' cost to the military community, including veterans, transitioning service members, and' +
+        ' military spouses and families.',
+    },
+    {
+      question:
+        'What do I need to start learning software development and how much is this going to cost?',
+      answer:
+        'First off, you will need access to a solid computer, a good browser (ex. Chrome), and a' +
```

Comment:
Instead of concatenated strings (`'some string' + 'another string'`), can you have ALL of the answers be within fragments?

Reply:
Ok sure. I used concatenated strings because the linter wouldn't allow the strings to be over 100 characters.